pmlc/target/x86/BUILD

```diff
@@ -7,7 +7,7 @@ load("//bzl:plaidml.bzl", "plaidml_cc_library")
 plaidml_cc_library(
     name = "x86",
     srcs = glob(["*.cc"]),
-    hdrs = glob(["*.h"]),
+    hdrs = glob(["*.h", "*.gz"]),
```

Comment:
I don't think this is necessary

Reply:
It is. It didb't work without it. If the file is not "exported" from the package it is not accessible from other packages.
This seems to be describing it.
https://stackoverflow.com/questions/43921911/how-to-resolve-bazel-undeclared-inclusions-error

Reply:
We can export the file if needs be in another way, but not through this mechanism.

Reply:
See: `exports_files`: https://docs.bazel.build/versions/master/be/functions.html#exports_files

Reply:
No need to do this anymore. All is moved to the target/x86 package.